src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -94,8 +94,6 @@ Assembly: 'SonarLint.2022, Version=6.0.0.0, Culture=neutral, PublicKeyToken=c5b6
 Relative path: 'SonarLint.2022.dll'
 
 Referenced assemblies:
-- 'Microsoft.CodeAnalysis, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
```

Comment:
We no longer have code that directly references the SonarAnalyzer callbacks, which means we don't need to reference the MS CodeAnalysis assemblies either.